test/functional/table-inheritance/single-table/database-option-inherited/database-option-inherited.ts

```diff
@@ -6,7 +6,8 @@ describe("table-inheritance > single-table > database-option-inherited", () => {
 
     let connections: Connection[];
     before(async () => connections = await createTestingConnections({
-        entities: [__dirname + "/entity/*{.js,.ts}"]
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        enabledDrivers: ["postgres", "cockroachdb", "mariadb", "mssql", "mysql", "sqlite", "sqljs"]
```

Comment:
I think on Oracle we have to unmount database first before creating another one - however creating more databases isn't always possible(e.g. it's blocked on official docker images) so we probably should just disable that test for oracle.